src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -70,8 +70,11 @@ reflected in a single DICOM tag for all possible aforementioned scan
 manipulations). See [here](https://lcni.uoregon.edu/kb-articles/kb-0003) and
 [here](https://github.com/neurolabusc/dcm_qa/tree/master/In/TotalReadoutTime)
 
-<sup>3</sup>We use the "FSL definition", i.e, the time between the center of the
```

Comment:
this was mentioned again under fieldmaps. I've moved that mention (after applying some improvements to the language) over here for completeness.